packaging/installer/methods/synology.md

```diff
@@ -59,6 +59,6 @@ installed. You'll have to do this manually:
 [ -x /etc/rc.netdata ] && /etc/rc.netdata start
 ```
 
-3. Make sure `/etc/rc.netdata` is executable: `chmod 0755 /etc/rc.netdata`.
+3. Make sure `/etc/rc.local` is executable: `chmod 0755 /etc/rc.local`.
```

Comment:
Just a minor adjustment if I may

remove the info about the executable from here https://github.com/netdata/netdata/pull/16980/files#diff-f390df6601297036281a13369310d7d97e6f4e7dc59588a2f08e8d8cdb86a1a3R53-R54


change it to something like

3. Make sure both `rc.netdata` and `rc.local` can be executed. Keep in mind if you already utilize `/etc/rc.local` you may not want to change its permissions, but to make this pipeline run you need to make sure they can be executed in series.
    ```
    chmod +x /etc/rc.local
    chmod 0755 /etc/rc.netdata
    ```

Reply:
@tkatsoulas I think it is ok as it is. You can make a PR with your suggestion if you want.